shoop/admin/modules/orders/views/refund.py

```diff
@@ -0,0 +1,249 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from __future__ import unicode_literals
+
+from django import forms
+from django.contrib import messages
+from django.core.urlresolvers import reverse
+from django.http.response import HttpResponseRedirect
+from django.utils.translation import ugettext as _
+from django.views.generic import UpdateView
+
+from shoop.admin.toolbar import PostActionButton, Toolbar, URLActionButton
+from shoop.admin.utils.urls import get_model_url
+from shoop.core.excs import (
+    NoRefundToCreateException, RefundExceedsAmountException
+)
+from shoop.core.models import Order
+
+
+class RefundForm(forms.Form):
+    line_number = forms.ChoiceField(
+        label=_("Line number"),
+        required=False,
+    )
+    quantity = forms.DecimalField(
+        required=False,
+        min_value=0,
+        initial=0,
+        label=_("Quantity"),
+    )
+    amount = forms.DecimalField(
+        required=False,
+        min_value=0,
+        initial=0,
+        label=_("Amount"),
+    )
+    restock_products = forms.BooleanField(
+        required=False,
+        initial=True,
+        label=_("Restock products")
+    )
```

Comment:
these have way too many line changes. I think these can be fitted in one row, all of these fields.